### ISSUE TITLE
LoadError on Julia nightly

### ISSUE THREAD

**Issue by pat-alt** [FluxML/IRTools.jl#105]
Hi there, 

I've been getting the following error on Julia nightly for the past few days (see [here](https://github.com/pat-alt/CounterfactualExplanations.jl/actions/runs/4164283371/jobs/7205722453#step:6:783) for full CI logs):

```julia
ERROR: LoadError: MethodError: no method matching Core.Compiler.IRCode(::Core.Compiler.InstructionStream, ::Core.Compiler.CFG, ::Vector{Core.LineInfoNode}, ::Vector{Any}, ::Vector{Expr}, ::Vector{Any})
```

Perhaps related to FluxML/Zygote.jl#287?

**Comment by CarloLucibello** [comment]
should be closed by #104

**Comment by pat-alt** [comment]
woah ... that was fast! Thanks
